docs/DeveloperGuide.adoc

```diff
@@ -358,12 +358,131 @@ image::UndoRedoActivityDiagram.png[width="650"]
 ** Cons: Requires dealing with commands that have already been undone: We must remember to skip these commands. Violates Single Responsibility Principle and Separation of Concerns as `HistoryManager` now needs to do two * different things.
 // end::undoredo[]
 
-// tag::dataencryption[]
-=== [Proposed] Data Encryption
 
-_{Explain here how the data encryption feature will be implemented}_
+// tag::remark[]
```

Comment:
Wish to see more sequence diagrams of how the command is invoked. Goes for all the other documentation. Please make sure this week's enhancement has some pictures. Remember, developer documentation has its uses (developer-as-maintainer and developer-as-user) and have to make it as readable as possible.